bazel/linux/defs.bzl

```diff
@@ -259,6 +259,16 @@ def _kernel_modules(ctx):
         modules = " ".join(modules),
     )
 
+    compilation_mode = ctx.var["COMPILATION_MODE"]
+    if compilation_mode == "fastbuild":
+        cflags = ""
+    elif compilation_mode == "opt":
+        cflags = ""
+    elif compilation_mode == "dbg":
+        cflags = "-O1 -fno-inline-functions-called-once"
+
```

Comment:
want to add an "else fail("compilation mode not supported?")"

Reply:
Added in case things change in the future. Bazel already enforces it:
```
ERROR: While parsing option -c george: Not a valid compilation mode: 'george' (should be fastbuild, dbg or opt)
```